resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/backend/minikube/Minikube.scala

```diff
@@ -120,7 +130,7 @@ private[spark] object Minikube extends Logging {
 
   def executeMinikube(action: String, args: String*): Seq[String] = {
     ProcessUtils.executeProcess(
-      Array("bash", "-c", s"minikube $action ${args.mkString(" ")}"),
+      Array("bash", "-c", s"MINIKUBE_IN_STYLE=true minikube $action ${args.mkString(" ")}"),
```

Comment:
For reviewers: Without this change,  the output of `minikube profile` is with the prefix `* ` .
(e.g. `* minikube` rather than `minikube`) in case stderr is not tty.